### ISSUE TITLE
Key Error: 2 when running numeric comparison

### ISSUE THREAD

**Issue by geobetts** [moj-analytical-services/splink#189]
I get a KeyError when including a numeric comparison column in settings - 'KeyError: 2'

The only fix I've managed is to write my own SQL expression.

Version: splink==1.0.4

**Comment by RobinL** [comment]
You need to set the `num_levels` key of the comparison column to equal the number of levels in your case statement.

**Comment by RobinL** [comment]
Ah - i've tracked it down with the more detailed example.  Yeah it's a bug:
https://github.com/moj-analytical-services/splink/blob/99d0a552925a7e92214941aa7b46acee3b2fcadf/splink/default_settings.py#L34. `:` rather than `=`

We will fix in the next release.

For now you can use one of the numeric options from the case statments library like:
```
from splink.case_statements import sql_gen_case_stmt_numeric_abs_3,sql_gen_case_stmt_numeric_perc_3
sql_gen_case_stmt_numeric_abs_3('my_col')
sql_gen_case_stmt_numeric_perc_3('my_col')
```

**Comment by RobinL** [comment]
Reopened so we remember to fix

**Comment by RobinL** [comment]
Closed by #190

**Comment by RobinL** [comment]
Closed by #190
